jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java

```diff
@@ -420,63 +390,27 @@ public void testCreateContainerBuilder_war()
             "/my/app/WEB-INF/classes/package/Other.class");
   }
 
-  @Test
-  public void testCreateContainerBuilder_defaultWebAppRoot()
-      throws URISyntaxException, IOException, InvalidImageReferenceException,
```

Comment:
This test wasn't even testing if the extraction paths start with `DEFAULT_WEB_APP_ROOT`. The test is identical to the one right above.